src/xcode/ENA/ENA/Source/Scenes/TicketValidation/TicketValidationCoordinator.swift

```diff
@@ -224,7 +224,11 @@ final class TicketValidationCoordinator {
 		alert.addAction(
 			UIAlertAction(
 				title: AppStrings.Common.alertActionOk,
-				style: .default
+				style: .default,
+				handler: { [weak self] _ in
+					self?.ticketValidation.cancel()
+					self?.navigationController.dismiss(animated: true)
+				}
```

Comment:
I just talked to Kerstin because I have doubts that it's correct to close the whole flow. Just blocking until it's clarified.